content/en/account_management/billing/custom_usage_reporting.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Custom usage reporting
+kind: documentation
+---
+
+## Overview
+
+Administrators can access the Custom Reporting tab from the Plan & Usage section in Datadog, then navigate to: 
+`Plan & Usage`--> `Custom Reporting`.
+
+This tool enables organizations to split their Datadog usage by up to 3 chosen tag keys and generates daily and monthly usage reports.
+
+The Custom Reporting page provides the following information and functionality:
+
+* It lists the existing tag keys that usage is being broken down by and provides the ability to change and add new ones (up to 3 tag keys).
+* It generates daily .tsv (tab separated values) files for most usage types.
+* It summarizes usage at the end of each month and surfaces both in the UI and as a .tsv download.
+
+_Note_: The following usage types are not supported in this tool yet. They will be added subsequently:
+
+* Indexed Log Events
+* Ingested Log Events
+* Analyzed Spans
+
+### Getting Started
+
+In order to start receiving daily data, an administrator will need to create a new report via the user interface. 
+
+{{< img src="account_management/billing/advanced-usage-reporting-01.png" alt="Getting Started" >}}
+
+The 'Applied Tags' section enables the following:
+
+* Entering up to three tag keys from a dropdown. The dropdown is pre-populated with existing tags on the account.
```

Comment:
`Entering up to three tag keys from a dropdown. The dropdown is pre-populated with existing tags on both the root account and any child organizations under the account.`

Add that last sentence.